src/pages/understand-stacks/stacking-using-CLI.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Stacking using the Stacks CLI
+description: Learn how use the Stacks CLI to participate in stacking
+experience: beginner
+duration: 10 minutes
+tags:
+  - guide
+images:
+  sm: /images/pages/stacking-rounded.svg
+---
+
+## Introduction
+
+!> The Stacking implementation is still in development and could change in the coming weeks
+
+In this guide, you'll learn how to use the Stacks CLI to participate in stacking. The CLI is a great way to quickly try out stacking on testnet. To integrate stacking into your application, see the [stacking integration guide](/stacks-blockchain/integrate-stacking).
+
+## Requirements
+
+First, you'll need to understand the [Stacking mechanism](/stacks-blockchain/stacking).
+
+You'll also need [NodeJS](https://nodejs.org/en/download/) `12.10.0` or higher to complete this tutorial. You can verify your installation by opening up your terminal and run the following command:
+
+```bash
+$ node --version
+```
+
+You will also need to install the Stacks CLI from NPM:
+
+```bash
+$ npm install @stacks/cli -g
+```
+
+## Generate A New Key Pair
```

Comment:
I'd apply the convnetion and indicate the step # in the heading: "Step 1: Generate an account"

- we adopted using "accounts" as opposed to wallet or key pair. I'd try to stay consistent. Introducing another term might reason confusion